### PR TITLE
3.40.0: Adds new SDK versions and contract files

### PR DESCRIPTION
**Pull request by kundadebdatta**
# Pull Request Template

## Description

This bumps the SDK GA version from `3.39.2` to `3.40.0` and preview version from `3.40.0-preview.2` to `3.41.0-preview.0`. Please see the below changes in the GA public contract.

image

Also highlighted below are the preview contract changes.

image
image

_[Note: To understand about the GA and preview versions styles, please take a look at our [versioning guidelines and principles](https://github.com/Azure/azure-cosmos-dotnet-v3/blob/master/docs/versioning.md)]_

### CODE REVIEW

**Review by Pilchie — DISMISSED** [review]

File: changelog.md [hunk 1]
```diff
@@ -13,6 +13,31 @@ Preview features are treated as a separate branch and will not be included in th
...
+- [4426](https://github.com/Azure/azure-cosmos-dotnet-v3/pull/4426) ChangeFeedProcessor: Fixes a bug properly when dealing with Legacy lease incremental documents that do not have a Mode property (#4426)
+- [4413](https://github.com/Azure/azure-cosmos-dotnet-v3/pull/4413) Query: Fixes Persisted continuationToken issue by turning off ODE by default (#4413)
+- [4459](https://github.com/Azure/azure-cosmos-dotnet-v3/pull/4459) Query: Fixes non streaming order by to use flag from query plan (#4459)
+- [4253](https://github.com/Azure/azure-cosmos-dotnet-v3/pull/4253) Query: Fixes LINQ Serialization CamelCase Naming Policy (#4253)
+- [4493](https://github.com/Azure/azure-cosmos-dotnet-v3/pull/4493) Query: Fixes non streaming OrderByCrossPartitionQueryPipelineStage to remove state and handle splits (#4493)
+
+#### Added
+- [4446](https://github.com/Azure/azure-cosmos-dotnet-v3/pull/4446) Query: Adds a new capability for non streaming order by in QueryFeatures (#4446)
+- [4447](https://github.com/Azure/azure-cosmos-dotnet-v3/pull/4447) FaultInjection: Adds SendDelay ServerErrorType (#4447)
```
Comment (kirankumarkolli) [inline changelog.md:34]: FaultInjection are not public features, lets exclude them

File: changelog.md [hunk 2]
```diff
@@ -13,6 +13,31 @@ Preview features are treated as a separate branch and will not be included in th
...
+
+#### Fixed
+- [4397](https://github.com/Azure/azure-cosmos-dotnet-v3/pull/4397) Query: Fixes ResponseMessage not parsing the IndexMetrics as text in latest sdk (#4397)
+- [4419](https://github.com/Azure/azure-cosmos-dotnet-v3/pull/4419) Query: Fixes bug in OrderByCrossPartitionQueryPipelineStage to ensure that errors in inner pipeline creation are bubbled up (#4419)
+- [4426](https://github.com/Azure/azure-cosmos-dotnet-v3/pull/4426) ChangeFeedProcessor: Fixes a bug properly when dealing with Legacy lease incremental documents that do not have a Mode property (#4426)
+- [4413](https://github.com/Azure/azure-cosmos-dotnet-v3/pull/4413) Query: Fixes Persisted continuationToken issue by turning off ODE by default (#4413)
+- [4459](https://github.com/Azure/azure-cosmos-dotnet-v3/pull/4459) Query: Fixes non streaming order by to use flag from query plan (#4459)
+- [4253](https://github.com/Azure/azure-cosmos-dotnet-v3/pull/4253) Query: Fixes LINQ Serialization CamelCase Naming Policy (#4253)
+- [4493](https://github.com/Azure/azure-cosmos-dotnet-v3/pull/4493) Query: Fixes non streaming OrderByCrossPartitionQueryPipelineStage to remove state and handle splits (#4493)
```
Comment (kirankumarkolli) [inline changelog.md:30]: Lets check with Aditya and Neil. 
Goal is to only list the ones that CX will observe.

**Review by kirankumarkolli — APPROVED** [review]

File: changelog.md [hunk 3]
```diff
@@ -13,6 +13,31 @@ Preview features are treated as a separate branch and will not be included in th
...
+#### Added
+- [4486](https://github.com/Azure/azure-cosmos-dotnet-v3/pull/4486) ContainerProperties: Enables Vector Embedding and Indexing Policy for Preview (#4486)
+
+### <a name="3.40.0"/> [3.40.0](https://www.nuget.org/packages/Microsoft.Azure.Cosmos/3.40.0) - 2024-05-17
+
+#### Fixed
+- [4397](https://github.com/Azure/azure-cosmos-dotnet-v3/pull/4397) Query: Fixes ResponseMessage not parsing the IndexMetrics as text in latest sdk (#4397)
+- [4419](https://github.com/Azure/azure-cosmos-dotnet-v3/pull/4419) Query: Fixes bug in OrderByCrossPartitionQueryPipelineStage to ensure that errors in inner pipeline creation are bubbled up (#4419)
+- [4426](https://github.com/Azure/azure-cosmos-dotnet-v3/pull/4426) ChangeFeedProcessor: Fixes a bug properly when dealing with Legacy lease incremental documents that do not have a Mode property (#4426)
+- [4413](https://github.com/Azure/azure-cosmos-dotnet-v3/pull/4413) Query: Fixes Persisted continuationToken issue by turning off ODE by default (#4413)
```
Comment (kirankumarkolli) [inline changelog.md:27]: Already shipped in 3.39.2

**Review by adityasa — APPROVED** [review]
:shipit:

File: changelog.md (hunk 1 above)
Comment (kundadebdatta) [reply changelog.md]: Removed.

File: changelog.md (hunk 3 above)
Comment (kundadebdatta) [reply changelog.md]: Removed.

**Review by adityasa — APPROVED** [review]
:shipit:

**Review by kirankumarkolli — APPROVED** [review]

**Review by sc978345 — APPROVED** [review]

**Pull request merged**